translate/convert/test_po2txt.py

```diff
@@ -45,6 +45,22 @@ def test_fuzzy_handling(self):
         assert self.po2txt(posource) == "Heading\n\nLyfteks"
         assert self.po2txt(posource, txttemplate) == "Heading\n\nLyfteks"
 
+    def test_obsolete_ignore(self):
+        """check that we handle fuzzy message correctly"""
+        txttemplate = "Heading\n\nBody text"
+        posource = '''
+msgid "Heading"
```

Comment:
Added its own test didn't want to overload a test that explicitly looking at obsolete units.